package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tidepool/viz",
-  "version": "1.3.0-weekly-to-bg-log.2",
+  "version": "1.3.0",
```

Comment:
Should this have a different version? @cbwebdevelopment

Reply:
Good catch. I rebased on top of develop.  So, Clint's commits are not part of this now.